test/user.test.js

```diff
@@ -205,6 +205,73 @@ describe('User', function() {
       });
     });
 
+    describe('Password validation with Maxlength', function() {
+      var pass72 = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfIt';
+      var pass73 = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfIt1';
+      var badPass = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfItXYZ1';
+
+      it('rejects passwords longer than 72 characters', function(done) {
+        try {
+          User.create({ email: 'b@c.com', password: pass73 }, function(err) {
+            if (err) return done(err);
+            assert.equal(err.statusCode, 422);
+            assert(false, 'Error should have been thrown');
+          });
+        } catch (e) {
+          done();
+        }
```

Comment:
I have tried few things here without using `try/catch`. I had to dig a bit more in `dao.js` Create function to see where it is not going but still couldn't find it. 
If I try anything other than try/catch, then the error is thrown: 
```
Error: Password too long:
```
The test works fine with try/catch, as when I put the proper password `pass72`, it fails.